src/bundler/bundle_v2.zig

```diff
@@ -9336,12 +9336,27 @@ const LinkerContext = struct {
                 );
 
                 switch (c.options.source_maps) {
-                    .external => {
+                    inline .external, .linked => |tag| {
```

Comment:
Does this need the `inline`?
```suggestion
                    .external, .linked => |tag| {
```

Reply:
this makes `tag` comptime, which is what it is being used in the `if`. i guess that isn't strictly needed for this use-case. i can remove.